ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.ts

```diff
@@ -0,0 +1,660 @@
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'aoe + bleed',
+          de: 'AoE + Blutung',
+          fr: 'AoE + Saignement',
+          ja: 'AOE + 出血',
+          ko: '전체 공격 + 도트',
+        },
+      },
+    },
+    // ---------------- second trash ----------------
+    {
+      id: 'ASS Infernal Pain',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7969', source: 'Sil\'dihn Dullahan', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'aoe + bleed',
+          de: 'AoE + Blutung',
+          fr: 'AoE + Saignement',
+          ja: 'AOE + 出血',
+          ko: '전체 공격 + 도트',
+        },
+      },
+    },
+    {
+      id: 'ASS Blighted Gloom',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7966', source: 'Sil\'dihn Dullahan', capture: false }),
+      response: Responses.getOut(),
+    },
+    {
+      id: 'ASS Infernal Weight',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '796B', source: 'Aqueduct Armor', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'aoe + In',
+        },
+      },
+    },
+    {
+      id: 'ASS Dominion Slash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '796A', source: 'Aqueduct Armor', capture: false }),
+      response: Responses.getBehind(),
+    },
+    // ---------------- Gladiator of Sil'dih ----------------
+    {
+      id: 'ASS Flash of Steel',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7671', source: 'Gladiator of Sil\'dih', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'ASS Rush of Might 1',
+      // Boss casts 765C (12.2s) and 765B (10.2s), twice
+      // Gladiator of Mirage casts 7659, 7658, 765A, these target the environment but are not reliable
+      // North
+      //                East               West
+      //   Line 1: (-34.14, -270.14) (-35.86, -270.14)
+      //   Line 2: (-39.45, -275.45) (-30.55, -275.45)
+      //   Line 3: (-44.75, -280.75) (-25.25, -280.75)
+      // South
+      //                East               West
+      //   Line 1: (-34.14, -271.86) (-35.86, -271.86)
+      //   Line 2: (-39.45, -266.55) (-30.55, -266.55)
+      //   Line 3: (-44.75, -261.25) (-25.25, -261.25)
+      // Center is at (-35, -271)
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '765C', source: 'Gladiator of Sil\'dih' }),
+      delaySeconds: 0.5,
+      promise: async (data, matches) => {
+        if (data.mightCasts.length === 2)
+          data.mightCasts = [];
+
+        // select the Hephaistoss with same source id
+        let gladiatorData = null;
+        gladiatorData = await callOverlayHandler({
```

Comment:
Can you remove this trigger from this PR so the rest of the patch can be merged and submit it separately?

Reply:
Sure!